guides/v2.2/cloud/configure/setup-cron-jobs.md

```diff
@@ -54,6 +54,9 @@ MAILTO=""
 * * * * *           cronrun
 ```
 
+{: .bs-callout-tip}
+Magento Cloud crons are not executed via crontab.
```

Comment:
```suggestion
You can use crontab to review the cron configuration for {{ site.data.var.ece }} projects;  however, the Magento Cloud platform does not use crontab to run cron jobs.
```

Reply:
**implemented**

Thank you, @meker12 👍